CHANGES.rst

```diff
@@ -61,6 +61,12 @@ astropy.modeling
 
 - Significant reorganization of the documentation. [#9078]
 - Add Tabular1D.inverse [#9083]
+- New function to generate new models from others by setting specific inputs
+  variable values to constants.
+- New function `fix_inputs' to generate new models from others by fixing
+  specific inputs variable values to constants.
+- New function to generate new models from others by setting specific inputs
+  variable values to constants.
```

Comment:
Something went wrong with the repeated entries. Also add a PR number.

Reply:
done

Reply:
there's one more duplicate entry to be removed.